specs/010-ssa-ir-validator/quickstart.md

```
@@ -0,0 +1,173 @@
+# Quickstart Guide: SSA-based IR Validator
+
+## Overview
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)